nservicebus/azure-servicebus/topologies/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Azure Service Bus Topologies
+summary: Azure Service Bus Topologies
+tags:
+- Azure
+- Cloud
+redirects:
+related:
+ - samples/azure/azure-service-bus
+reviewed: 2016-04-12
+---
+
+Messaging topology is an arrangement of the various messaging entities.
+Azure Service Bus transport operates on a topology created on the broker. Messaging entities participating in topology are queues, topics, subscriptions, and rules. Combined in a specific way they form a topology. 
+
+Topology handles message exchange between endpoints by creating and configuring Azure Service Bus entities.  
+
```

Comment:
> Messaging topology is an arrangement of the various messaging entities.

> Azure Service Bus transport operates on a topology created on the broker. Messaging entities participating in topology are queues, topics, subscriptions, and rules. Combined in a specific way they form a topology. 
 
> Topology handles message exchange between endpoints by creating and configuring Azure Service Bus entities.

Messaging topology is a specific arrangement of the messaging entities, such as queues, topics, subscriptions, and rules. 

Azure Service Bus transport operates on a topology created on the broker. Topology handles exchanging messages between endpoints, by creating and configuring Azure Service Bus entities.